src/features/events/components/EventParticipansTab/EventParticipantsList.tsx

```diff
@@ -0,0 +1,173 @@
+import { FC } from 'react';
```

Comment:
The module (file) that this code is in is misspelled. I also don't think it should be inside a folder at all, just put it right at `src/features/events/components/EventParticipantsList.tsx`.

Reply:
Please also remove the folder `EventParticipantsList` and put `AddPersonButton` directly in `components`. I realize this is not your code originally but I didn't realize when I reviewed #1267 that it was in this directory.